app/routes/planet_routes.py

```diff
@@ -13,8 +43,33 @@ def get_all_planets():
 @planets_bp.get("/<planet_id>")
 def get_one_planet(planet_id):
     planet = validate_planet(planet_id)
-    return planet.to_dict(),200
-    
+    return planet.to_dict()
+
+@planets_bp.get("/count")
+def get_count():
+    query = db.session.query(Planet.name).count()
```

Comment:
This `session.query()` syntax is considered legacy in the current SqlAlchemy, and shouldn't be used any longer. When researching how to do operations, always be suer that any resources apply to the version of the library (SqlAlchemy) in use. The current SqlAlchemy approach would be as follows
```py
    # need to import the func module from sqlalchemy before this
    query = db.select(func.count(Planet.id))
    result = db.session.scalar(query)  # the count is in result
```

It's longer (though it could still be done in one line), but it more cleanly separates the construction of the query from running the query.